spec/controllers/shops_controller_spec..rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe ShopsController, type: :controller do
+  describe "signup page" do
+    context "when user requests signup url" do
+      it "returns a 200 status code" do
+        get :new
+        expect(response.status).to eql(200)
+      end
+
+      it "loads the signup page" do
+        get :new
```

Comment:
Two `get` requests were made for similar tests. This can be stripped down to one request for optimization